common-util/List/ListCommon.jsx

```diff
@@ -82,7 +82,7 @@ DependencyLabel.defaultProps = { type: 'component' };
 
 export const RegisterMessage = ({ handleCancel }) => (
```

Comment:
Mint should only be used on original NFT creation, not sure where these messages below come into play. Please check